packages/api-v2/babel.config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@jim-deriv I dont think that this is needed as we are importing the API-V2 in different packages and build and transpile it in the consumer package.
Same goes for api, components and shared packages.
For that we need to exclude the mentioned packages in the [workspaces](https://github.com/deriv-com/deriv-app/pull/17076/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R4) section of root package.json

Reply:
Hi @ali-hosseini-deriv, If it's not present, test cases won't run.
I think this is related to the deprecation of `babel-polyfill` and introduction of `core-js` which would be used as follows. That change was most likely introduced as a breaking change in one of the `jest` versions.

`{
    "presets": [
        [
            "@babel/preset-env",
            {
                "useBuiltIns": "entry",
                "corejs": "core-js@2",
                "debug": false
            }
        ]
    ]
}`

<img width="1174" alt="image" src="https://github.com/user-attachments/assets/a6430418-a977-41b2-b810-e96a945d579d">

Reply:
I've listed only the `packages` that should be built @ali-hosseini-deriv